decoimpact/business/entities/rules/string_parser_utils.py

```diff
@@ -84,6 +84,12 @@ def type_of_classification(class_val) -> str:
         if ":" in class_val:
             str_range_to_list(class_val)
             return "range"
+        if ">=" in class_val:
+            read_str_comparison(class_val, ">=")
+            return "larger_equal"
+        if "<=" in class_val:
+            read_str_comparison(class_val, "<=")
+            return "smaller_equal"
```

Comment:
Sharp. Needed to change the order in this file, because the code should not first check on > included before >=.. that goes wrong. I will update the classification_rule